llvm/test/Transforms/LoopVectorize/invariant-store-vectorization.ll

```diff
@@ -13,9 +13,6 @@ target datalayout = "e-p:64:64:64-i1:8:8-i8:8:8-i16:16:16-i32:32:32-i64:64:64-f3
 ; address.
 
 
-; memory check is found.conflict = b[max(n-1,1)] > a && (ptr a)+1 > (ptr b)
```

Comment:
Why is this removed? It looks like there is a memcheck below.

Reply:
That's a good question! I wasn't sure what to do here, because the exact contents of found.conflict depend upon whether you're trying to hoist the runtime memory checks or not. Perhaps it was just lazy of me to remove it - I'll update the comment to reflect what's it actually now checking.